r-package/grf/DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: grf
 Title: Generalized Random Forests
-Version: 1.2.0.0
+Version: 2.0
```

Comment:
Should this be 2.0.0.0 (not sure if CRAN requires this)?

Reply:
Thanks, yes it should be `2.0.0`! the final digit was just something we tacked onto the github version so we could easily distinguish the development version from the CRAN version, since this development version lingered around for a year